ReactiveCocoa/Swift/Bag.swift

```diff
@@ -18,7 +18,7 @@ internal final class RemovalToken {
 
 /// An unordered, non-unique collection of values of type T.
 internal struct Bag<T> {
-	private var elements: [BagElement<T>] = []
```

Comment:
I think the alternative is clearer: it separates the type definition from the value.